src/init.cr

```diff
@@ -6,38 +6,13 @@ class LuckyCli::Init
   end
 
   def run
-    project_name = name
```

Comment:
The  `name` private method should also be removed and maybe it should error if any args are provided?

Reply:
From the look of it, there can still be an argument for the task stuff. We could move that out of `init` and in to something else. So right now `lucky init tasks.cr` generates a new task file. We could do something like `lucky generate.task` or something?

Reply:
I like the idea of a different task. Maybe `lucky gen.tasks_file` to is it clear it is not a task